en/tutorials-and-examples/cms/articles-controller.rst

```diff
@@ -33,8 +33,8 @@ look like this::
     {
         public function index()
         {
-            $this->loadComponent('Paginator');
-            $articles = $this->Paginator->paginate($this->Articles->find());
+            $this->set('articles', $this->paginate($this->Articles));
+            $articles = $this->Articles->find();
             $this->set(compact('articles'));
```

Comment:
Currently this would execute the query twice (once with `$this->paginate($this->Articles)` and once with `$this->Articles->find()`)

I would personally prefer just a simple
```
$articles = $this->Articles->find();
$this->set(compact('articles'));
```
then its clear what is executing the query and what is setting the view variable

Reply:
But there would be no pagination with that usage, correct?
Is that the same as the following?
```php
            $articles = $this->paginate($this->Articles);
            $this->set(compact('articles'));
```

Reply:
yup, my bad. your version is here is correct.